chcemvediet/apps/inforequests/constants.py

```diff
@@ -1 +1,3 @@
 INFOREQUESTS_PER_PAGE = 50
+DAYS_TO_CLOSE_INFOREQUEST = 100
```

Comment:
Nazov konstatny `DAYS_TO_CLOSE_INFOREQUEST` nie je uplne presny, pretoze tento pocet dni sa aplikuje iba na expirovane deadliny. Tj. ak inforequest skonci akciou, ktora nema ziaden deadline, tak za zavrie ihned.

Nazov by mal vyjadovat nieco ako "pocet dni po ktorych za infoziadost zatvori po expirovani deadlinu", ale samozrejnie nie v tak vela slovach. Skus nieco vymysliet.

Reply:
A co tak nazov `DAYS_TO_FORCE_CLOSE_INFOREQUEST`? Podla mna ani ten povodny nie je zly, kedze sa jedna o zlozitejsi pripad kedy sa moze infoziadost uzatvorit, radsej by som pri definicii konstanty pridal komentar na co sluzi: "pocet dni po ktorych sa infoziadost zatvori po expirovani deadlinu".

Reply:
@viliambalaz Ok. Nechaj teda povodny nazov a napis pred neho vysvetlujuci komentar.

Reply:
@viliambalaz Pls, vzdy pred komentar daj prazdny riadok:

```

INFOREQUESTS_PER_PAGE = 50

# Number of days after which inforequest can be mark as closed, considering to branch deadline.
DAYS_TO_CLOSE_INFOREQUEST = 100 
```

Reply:
@viliambalaz Pls namiesto "considering to branch deadline" skor napis "after missing all deadlines"

A analogicky komentar napis pls aj ku `DAYS_TO_PUBLISH_INFOREQUEST`